### ISSUE TITLE
RecursionError when installing MoviePy

### ISSUE THREAD

**Issue by roniemartinez** [python-poetry/poetry#2329]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: Mac OS X 10.15.4
- **Poetry version**: 1.0.5

## Issue
```console
% poetry add moviepy -E optional    
Using version ^1.0.2 for moviepy

Updating dependencies
Resolving dependencies... (17.9s)

[RecursionError]
maximum recursion depth exceeded
```

**Comment by isuckatprogram** [comment]
Is there a fix to this?

**Comment by abn** [comment]
Fixed in `1.1.0b2`.
